testsuite/forge/src/main.rs

```diff
@@ -34,7 +34,8 @@ struct Args {
         default_value = "60"
     )]
     duration: u64,
-    // TODO, need to remove these args once we can load from vault
+    // these keys are used when testing against local swarm only
+    // testnet backend fetches these programatically via vault
```

Comment:
This isn't needed for local swarm testing, lets just remove these arguments completely.